CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.3.3] - 2020-12-02
+
```

Comment:
Shit - the dependency bump didn't have any changelog :(

Reply:
ok lets made it dirty here